src/polycubed/src/base_cube.cpp

```diff
@@ -383,7 +383,7 @@ nlohmann::json BaseCube::to_json() const {
 
   j["name"] = name_;
   j["uuid"] = uuid_.str();
-  j["service"] = service_name_;
+  j["service-name"] = service_name_;
```

Comment:
Can we use "underscore" instead of "hyphen" in `service-name` to keep the uniformity intact?

Reply:
`service-name` is a json field exposed to the user while `service_name` is a variable.
In the datamodel I also used `service-name` what I think is better for user interaction.

Reply:
@acloudiator I agree with Mauricio. For users, "service-name" looks better. Internally in the framework we tend to use "_" more tna "-", hence "service_name"; so this is the reason of this small difference.